.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -2479,17 +2549,20 @@ public class ConversationSessionStartedUpdate : ConversationUpdate, IJsonModel<C
         public static bool operator !=(ConversationStatus left, ConversationStatus right);
         public override readonly string ToString();
     }
-    public abstract class ConversationStatusDetails : IJsonModel<ConversationStatusDetails>, IPersistableModel<ConversationStatusDetails> {
+    public class ConversationStatusDetails : IJsonModel<ConversationStatusDetails>, IPersistableModel<ConversationStatusDetails> {
+        public string ErrorCode { get; }
+        public string ErrorType { get; }
+        public ConversationIncompleteReason? IncompleteReason { get; }
         public ConversationStatus StatusKind { get; }
         public static explicit operator ConversationStatusDetails(ClientResult result);
         public static implicit operator BinaryContent(ConversationStatusDetails conversationStatusDetails);
     }
     public class ConversationTokenUsage : IJsonModel<ConversationTokenUsage>, IPersistableModel<ConversationTokenUsage> {
+        public int InputTokenCount { get; }
         public ConversationInputTokenUsageDetails InputTokenDetails { get; }
-        public int InputTokens { get; }
+        public int OutputTokenCount { get; }
         public ConversationOutputTokenUsageDetails OutputTokenDetails { get; }
-        public int OutputTokens { get; }
-        public int TotalTokens { get; }
+        public int? TotalTokens { get; }
```

Comment:
Should this be "TotalTokenCount" and not optional?

Reply:
good catch; I had to customize all of these in /realtime and missed total at the top level. I'll add a few asserts into the tests while changing this, too, as it looks like `Usage` wasn't ever being explicitly exercised.